File: tinhoctre/thtbg2023b_mid.cpp
// https://tinhoctre.vn/problem/thtbg2023b_mid
#include<iostream>
#include<bits/stdc++.h>

using namespace std;

int main() {
    int a, b;
    cin >> a >> b;
    long long int min1 = INT_MAX;
    long long int min2 = INT_MAX;
    for (long long int i = a + 1; i < b; i++) {
        if (abs(a * i - b * i) < min1) {
            min1 = abs(a * i - b * i);
            min2 = i;
        }
    }
    cout << min2;
}

File: tinhoctre/thtbg2023b_mid.py
#https://tinhoctre.vn/problem/thtbg2023b_mid

input = input()
input = input.split(" ")
a = int(input[0])
b = int(input[1])
min = [1e+9, 1e+9]

for i in range(a + 1, b):
    if abs(i * a - i * b) < min[0]:
        min[0] = abs(i * a - i * b)
        min[1] = i

print(min[1])